src/Relationships/HasMany.php

```diff
@@ -36,6 +36,12 @@
 	/** @var IEntity[] */
 	protected $toRemove = [];
 
+	/** @var IEntity[] */
+	protected $added = [];
```

Comment:
uplne nechapu, proc znovunacteni z db by to nevythalo, kvuli relationship cache?